src/components/inspector/tooltip.vue

```diff
@@ -129,6 +129,11 @@ export default {
       variant: '',
     };
   },
+  computed: {
+    buttonType() {
+      return this.$attrs.selectedControl.config.event;
```

Comment:
Is there ever a case where part of this entire path (`this.$attrs.selectedControl.config.event`) may cease to exist and then return an undefined error in the console? Just want to make sure we have some error handling here. Might be good to use the [Lodash _.get method](https://lodash.com/docs/#get) here to prevent any errors in the future.

Reply:
Something similar to `_.get(this.$attrs, 'selectedControl.config.event')` should work.